sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -5988,7 +5988,6 @@ pi_result piEventGetProfilingInfo(pi_event Event, pi_profiling_info ParamName,
   }
   case PI_PROFILING_INFO_COMMAND_QUEUED:
   case PI_PROFILING_INFO_COMMAND_SUBMIT:
-    // TODO: Support these when Level Zero supported is added.
     return ReturnValue(uint64_t{0});
```

Comment:
Returning "0" is still not the right behavior. We should use `zeDeviceGetGlobalTimestamps` to record the time when commands were physically submitted to the device (inside plugin). I am OK if you just add a TODO comment and not fix it in this PR since that will still be unused (btw, also mention this that there are currently no users of this).